### DISCUSSION TITLE
Enhance hover styles

### DISCUSSION THREAD

**Discussion by Febakke** [digdir/designsystemet#2466] [Feature requests]
After discusion with Brreg we decided to investigate some of our discrete hover states. 
participants:@camulos @tvs-brreg @LeneSundsbo @mrosvik @Febakke 

Here are the three components we have identified now. Cards, accordion and button. 

<img width="868" alt="Skjermbilde 2024-02-29 kl  14 30 19" src="https://github.com/digdir/designsystemet/assets/33222679/3d1b6aad-ed4e-4494-b522-0ac8d8fa2915">

<img width="828" alt="Skjermbilde 2024-02-29 kl  14 30 56" src="https://github.com/digdir/designsystemet/assets/33222679/d7031c7a-fbbe-49f9-9234-cc87de5d593f">

<img width="567" alt="Skjermbilde 2024-02-29 kl  14 31 13" src="https://github.com/digdir/designsystemet/assets/33222679/836cf10b-1e15-4f1f-a2c7-c19e42e676b9">

Solutions that were discussed:
- Add underline on text in button while hovering ([Oslo Origo solved their button this way](https://punkt.oslo.kommune.no/5.1/komponenter/button/))
- Add stronger border to card (this may apply to accordion as well)

```[tasklist]
### Tasks
- [ ] Investigate different solutions
- [ ] Enhance hover states in figma
- [ ] Enhance hover states in react 
```

**Comment by mrosvik** [comment]
Liten research på hva andre gjør når det gjelder hover på `Button`. De fleste jeg sjekket innom bruker altså en antydning mørkere eller lysere farge for hover. 

Noen gjør knappen større og noen inverterer den. Av de nederste heller vi mot Oslo Kommune sin variant fordi den beholder fargen, men legger på en strek i tillegg. Invertering kan være uheldig fordi teksten endrer seg betydelig og forskjellen mellom primær og sekundærknapp forsvinner. 

Vi vil uansett gjøre mer research før vi evt. endrer dagens oppførsel på `Button`. 

<img width="1007" alt="image" src="https://github.com/digdir/designsystemet/assets/1464915/9b10a895-cb10-4f5f-9cd1-933177336305">
